fluent/src/scope.js

```diff
@@ -6,13 +6,17 @@ export default class Scope {
     insideTermReference = false,
     dirty = new WeakSet()
   ) {
+    // The bundle for which the given resolution is happening.
```

Comment:
Should these be exposed through esdocs?

One thing about `Scope` on esdocs is that it's the only class there that's not `FluentFoo`. Is that good or bad?

Reply:
> Should these be exposed through esdocs?

Fixed.

> One thing about Scope on esdocs is that it's the only class there that's not FluentFoo. Is that good or bad?

It's also the only class that is not part of the public API and it's not exposed through `index.js`. I think it's OK to keep the current name.